src/main/java/com/naveenautomation/Pages/MonitorsPage.java

```diff
@@ -0,0 +1,46 @@
+package com.naveenautomation.Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import com.naveenautomation.Browsers.ProxyDriver;
+
+public class MonitorsPage extends Page{
+
+	
+	private static final String PAGE_URL="/category&path=25_28";
```

Comment:
we can't have page with &path=25_28"

Reply:
This is page url and it is unique and required to load the page.

Reply:
&path=25_28 is specific to a product and not to page , remove this

Reply:
I have checked it again. &path=25_28 is not referring to any product .It is specific to monitors page.

Reply:
I have now defined a method which will verify the static part of page url.

Reply:
dynamic part of url removed